tools/pre-commit

```diff
@@ -52,11 +52,10 @@ else
 	
 	for file in ${files}; do
 		# Validate each file using
-		git show :$file | $reserialize -f -
+		git show :$file | $reserialize -r -f -
```

Comment:
Tijdens pre-commit wil je niet bestanden aanpassen, die aanpassingen gaan standaard ook niet mee in de commit die gedaan wordt (ze moeten namelijk dan nog via git add toegevoegd worden, wat je niet automatisch wil omdat je dan meteen het hele bestand meepakt terwijl dit soms onwenselijk kan zijn)